scripts/build-and-provide-package

```diff
@@ -540,6 +564,23 @@ EOF
     echo "USENETWORK=yes" >> "$pbuilderrc"
   fi
 
+  # add libeatmydata to pbuilderrc
+  if [ "${USE_EATMYDATA:-}" = true ] ; then
+     echo 'EXTRAPACKAGES="$EXTRAPACKAGES eatmydata"' >> "$pbuilderrc"
+     echo 'if [ -z "$LD_PRELOAD" ]; then' >> "$pbuilderrc"
+     echo '   LD_PRELOAD=libeatmydata.so' >> "$pbuilderrc"
+     echo 'else' >> "$pbuilderrc"
+     echo '   LD_PRELOAD="$LD_PRELOAD":libeatmydata.so' >> "$pbuilderrc"
```

Comment:
can use `LD_PRELOAD="${LD_PRELOAD:+$LD_PRELOAD:}libeatmydata.so"` instead of the if thing